Makefile.depends

```diff
@@ -15,7 +15,7 @@ $(call depends,modules/benchmark): | .install/base/db .install/base/logger .inst
 $(call depends,modules/data.atmosphere): | .install/base/db .install/base/logger .install/base/remote .install/base/utils
 $(call depends,modules/data.hydrology): | .install/base/logger .install/base/utils
 $(call depends,modules/data.land): | .install/modules/benchmark .install/modules/data.atmosphere .install/base/db .install/base/logger .install/base/remote .install/base/utils .install/base/visualization .install/base/settings
-$(call depends,modules/data.remote): | .install/base/db .install/base/utils .install/base/logger .install/base/remote
+$(call depends,modules/data.remote): | .install/base/db .install/base/utils .install/base/logger .install/base/remote .install/modules/data.land
```

Comment:
This creates a circular dependency.  What was changed that makes data.remote now depend on data.land?

Reply:
When I added the soilgrids_soc_extract function from Cherry, GitHub asked me to add data.land as dependency.

Reply:
I will move it into data.land package.